src/Umbraco.Infrastructure/Serialization/JsonObjectConverter.cs

```diff
@@ -101,8 +101,6 @@ public override void Write(Utf8JsonWriter writer, object value, JsonSerializerOp
             JsonTokenType.Number when reader.TryGetInt32(out int i) => i,
             JsonTokenType.Number when reader.TryGetInt64(out long l) => l,
             JsonTokenType.Number => reader.GetDouble(),
-            JsonTokenType.String when reader.TryGetDateTimeOffset(out DateTimeOffset datetime) => datetime,
```

Comment:
Could you add a comment please justifying why `DateTime` and `DateTimeOffset` are no longer handled here please (just to prevent a future person noticing that we aren't handling all primitive conversions here and wondering why or adding it back).